### ISSUE TITLE
Save separate sessions

### ISSUE THREAD

**Issue by Chronolegionnaire** [luetkemj/aglet-timekeeper#26]
It would be great to be able to have "sessions" for dm's that run multiple games that way we can just click a few different sessions and pull up the time/date of the specific session rather than having it written down elsewhere and changing the date/time accordingly.

**Comment by luetkemj** [comment]
Yeah, that was always a plan to add. Trying to balance all these great ideas with keeping things simple and intuitive. I'll see what I can do.
